styles/style.css

```diff
@@ -0,0 +1,123 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+
+* {
+    font-family: 'Roboto', sans-serif;
+    max-width: 1440px;
+    margin: 0 auto;
+}
+
+h1 {
+    display: flex;
+    justify-content: center;
+    width: 306px;
+    height: 47px;
+    margin: 120px 567px 0 567px;
+}
+
+hr {
+    border: none;
+    background-color: #F5BD41;
+    height: 4px;
+    width: 100px;
+    margin: 20px 670px;
+}
+
+.main {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    margin-top: 60px;
+    margin-bottom: 100px;
+}
+
+.box {
+    width: 296px;
+    height: 176px;
+}
+
+.headphones img {
```

Comment:
add img wrapper for all images wuth property width:100px, height, 100px, then for images you can use property height auto and you didn't  need to describe five different classes

Reply:
but every img has its own size and in Figma there diffirent sizes for each img